components/Button/index.js

```diff
@@ -0,0 +1,21 @@
+import styled from "styled-components";
+
+export const Button = ({ setProduct, produkt, productToBuy }) => {
```

Comment:
Here you can choose one prop to get the same result.
```suggestion
export const Button = ({ setProduct, productToBuy }) => {
```